CHANGELOG.md

```diff
@@ -24,6 +53,10 @@
 * **Spec: Allow nested struct fields in `SchemaDatasetFacet`** [`#2548`](https://github.com/OpenLineage/OpenLineage/pull/2548) [@dolfinus](https://github.com/dolfinus)
     *Allows nested fields support to `SchemaDatasetFacet`.*
 
+### Removed
+* **client/java: [Client] Remove runId from Kafka messageKey** [`#2615`](https://github.com/OpenLineage/OpenLineage/pull/2615) [@dolfinus](https://github.com/dolfinus)  
```

Comment:
This item can be removed

Reply:
I mean removed. Option `localServerId` -> `messageKey` was removed in #2571, not in #2615.

Reply:
Thanks for catching this, @dolfinus